citi-onboarding/DADe-onepage
# feature(model-contato): Created contato model, controller and route

## What I did
- Created model, controller and route for contato
- Added `nocreate: true` property  to List model
- Added `nodelete: true` property  to List model

## How to test
1. Open the terminal and go to the folder of this project

2. In terminal:
   1. git checkout develop
   2. git pull
   3. git checkout feature/model-contato
   4. git pull origin feature/model-contato
   5. npm install
   6. node index.js

3. In your browser go to http://localhost:3001/admin , and check if the contato list is appearing, then open the "Contatos" list to check if there is a item and if you can edit it (Fields: Instagram, twitter, telegram, email, whatsapp). Also check if you are unable to create a new item to the list (nocreate:true) and also unable to delete the existing item (nodelete:true).

4. In your browser go to http://localhost:3001/api/contatos, and check if the component informations are there showing it's details.
